CB-11112 Added Appium documentation

alsorokin opened the pull request:
https://issues.apache.org/jira/browse/CB-11112

rakatyal commented on APPIUM.md:
```diff
@@ -0,0 +1,64 @@
+
+If running on simulator, no need to launch it manually - Appium will do it automatically.
+
+Command sample to run tests on the "iPhone 6" simulator with iOS 8.4:
```
Should we include an example to run it on a device for android/ios?

rakatyal commented on APPIUM.md:
```diff
@@ -0,0 +1,64 @@
+
+### Running on Android
+
+Cordova Appium tests for Android can be ran on either real device or an emulator. If you're running tests on the emulator you will need to launch it manually before the test run. Please also note that Appium tests are written with Android API 19 (KitKat) and 21 (Lollipop) in mind. They're not yet tested on higher versions of Android OS.
```
Are there any OS/software requirements to run Appium tests? Like min OSX version/Xcode version or something?

rakatyal commented:
Minor comments. Otherwise LGTM.

riknoll commented on APPIUM.md:
```diff
@@ -0,0 +1,64 @@
+### Running on Android
+
+Cordova Appium tests for Android can be ran on either real device or an emulator. If you're running tests on the emulator you will need to launch it manually before the test run. Please also note that Appium tests are written with Android API 19 (KitKat) and 21 (Lollipop) in mind. They're not yet tested on higher versions of Android OS.
+
```
Grammar nitpick:
Cordova Appium tests for Android can be *run* on either *a* real device or an emulator.

*They* have not yet *been* tested on higher versions of Android

alsorokin replied:
Thanks a lot for this. I clearly need to practice English more.

riknoll commented on APPIUM.md:
````diff
@@ -0,0 +1,64 @@
+Clone medic repository by running
+
+``` shell
+ > git clone https://github.com/apache/cordova-medic
````
You also need to run `npm install` in the repo

riknoll commented:
Very helpful guide! I was able to get the tests running on my machine (for Android, didn't try iOS). One thing to note is that it seems like the tests are very dependent on the apps installed on the device running them and it might be helpful to document that. For example, the tests for camera that use PHOTOLIBRARY as a source do not work with the gallery app that comes with the VS Android emulator (they just time out).

alsorokin commented:
@riknoll @rakatyal Addressed review notes, please see https://github.com/apache/cordova-medic/pull/93/commits/0e7ff36d9a53d72e361eb86574a224f904e535e6

rakatyal commented on APPIUM.md:
```diff
@@ -58,7 +71,17 @@ Command sample to run tests on the "iPhone 6" simulator with iOS 8.4:
 [brew]:            http://brew.sh/
 [plugin_camera]:   https://github.com/apache/cordova-plugin-camera
 [plugin_contacts]: https://github.com/apache/cordova-plugin-contacts
+[appium_docs]:     http://appium.io/getting-started.html
```
There is already [appium_docs] defined so consider renaming it. Also you may directly point it to 'http://appium.io/getting-started.html#requirements'.

alsorokin replied:
Nice catch. Fixed

rakatyal commented:
LGTM.

riknoll commented:
LGTM

alsorokin commented:
Thanks guys. Merging
